azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/CallCompositeBuilder.java

```diff
@@ -39,6 +44,18 @@ public CallCompositeBuilder localization(final CallCompositeLocalizationOptions
         return this;
     }
 
+    public CallCompositeBuilder setCallScreenOrientation(
```

Comment:
Let's stick to existing naming format without 'set' - callScreenOrientation(...)
Same for setup screen

Reply:
sure. updating it.